src/main/java/com/epam/controller/IndexController.java

```diff
@@ -1,19 +1,34 @@
 package com.epam.controller;
 
+import com.epam.model.User;
+import com.epam.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
 
 @Controller
+@RequestMapping(path="/demo")
 public class IndexController {
-    @GetMapping("/")
-    public ModelAndView index() {
-        Map<String, String> model = new HashMap<>();
-        model.put("name", "Alexey");
-        return new ModelAndView("index", model);
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping(path="/add")
+    public @ResponseBody String addNewUser (@RequestParam String name) {
+
+
+        User n = new User();
+        n.setFirstName(name);
+        userRepository.save(n);
+        return "Saved";
```

Comment:
You don't have to return a String to describe status of the request, it's better to use response codes instead. jQuery is able to differentiate success codes (2xx) from errors (4xx, 5xx).